### PR TITLE
✨  Consultancy hero section#176

### LINKED ISSUE #176: 🎨 Create the consultancy hero section
#  Create the consultancy hero section 
---

Quick links: [Summary](#summary) | [Description](#description) | [Technical-Analysis](#Technical-Analysis) | [Acceptance-Criteria](#Acceptance-Criteria)

---

### Summary

As a user, I want a nice hero section on the consultancy page that captures my attention and introduces the page content. Reuse the text-content component created in #20, and the reusable slider button in  #24. I want a good experience on the web and phone.

Design:

consultancy hero section

Link to full design:

```
https://xd.adobe.com/view/b98caf27-6549-49e5-8ed8-c5234788df61-ecdf/screen/cc35e19a-24f2-42ce-8803-ee2942839528/
```
 
### Technical Analysis

This feature can be found in libs/features/pages/consultancy-page inside a components folder. The component's name is consultancy-hero-section.

### Acceptance Criteria

1. When rendered the component is pixel perfect compared to the design above.
2. The component is clean and does not have more than 30 lines
3. The component SCSS uses CSS variables for colour, font, ... Connected to the global style.scss
5. The design is responsive for screen, tablet and phone

### PR DESCRIPTION
**Pull request by Marionwanyama23**
# Description

This pull request contains a hero section on the consultancy page that captures the user's attention and introduces the page content. It reuses the reusable hero text component #20  and the reusable slider button in #24. 

Fixes #176 

## Type of change

- [x] New feature (non-breaking change which adds functionality)

## Screenshot 
Screenshot from 2023-09-08 10-51-12

## Checklist

- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my code
- [x] I have commented my code, particularly in hard-to-understand areas
- [x] I have made corresponding changes to the documentation
- [x] My changes generate no new warnings
- [x] I have added tests that prove my fix is effective or that my feature works
- [x] New and existing unit tests pass locally with my changes
- [x] Any dependent changes have been merged and published in downstream modules

### CODE REVIEW

File: libs/features/pages/consultancy-page/src/lib/components/consultancy-hero-section/consultancy-hero-section.component.scss [hunk 1]
```diff
@@ -0,0 +1,128 @@
...
+    display: grid;
+    gap: 20px;
+    margin-left: 260px;
+    margin-top: 200px;
+    position: relative;
+
+    .title {
+      width: 915px;
+      height: 106px;
+      font-family: DMSans-Medium;
```
Comment (AnnaL001) [inline libs/features/pages/consultancy-page/src/lib/components/consultancy-hero-section/consultancy-hero-section.component.scss:18]: For reuse of fonts, use CSS variables specified in elements/theming/_variables.scss for specifying font family

File: libs/features/pages/consultancy-page/src/lib/components/consultancy-hero-section/consultancy-hero-section.component.scss [hunk 2]
```diff
@@ -0,0 +1,128 @@
...
+      font-size: 82px;
+      color: var(--elewa-website-color-primary-black);
+    }
+
+    .description {
+      width: 741px;
+      height: 197px;
+      font-family: DMSans-Regular;
```
Comment (AnnaL001) [inline libs/features/pages/consultancy-page/src/lib/components/consultancy-hero-section/consultancy-hero-section.component.scss:26]: Same here

File: libs/features/pages/consultancy-page/src/lib/consultancy-page.module.ts [hunk 3]
```diff
@@ -4,9 +4,13 @@ import { CommonModule } from '@angular/common';
 import { ConsultancyRoutingModule } from './consultancy.routing';
 
 import { ConsultancyPageComponent } from './main/consultancy-page/consultancy-page.component';
+import { ConsultancyHeroSectionComponent } from './components/consultancy-hero-section/consultancy-hero-section.component';
+import { ButtonsModule } from '@elewa-website/elements/layout/buttons';
+import { AppHeaderModule } from '@elewa-website/elements/layout/header';
```
Comment (AnnaL001) [inline libs/features/pages/consultancy-page/src/lib/consultancy-page.module.ts:10]: Kindly, observe the order of imports; elements imports should appear before local imports

**Review by AnnaL001 — CHANGES_REQUESTED** [review]
Nicely done, just a couple of changes

File: libs/features/pages/consultancy-page/src/lib/components/consultancy-hero-section/consultancy-hero-section.component.scss (hunk 1 above)
Comment (Marionwanyama23) [reply libs/features/pages/consultancy-page/src/lib/components/consultancy-hero-section/consultancy-hero-section.component.scss]: I have made the changes

**Comment by LemmyMwaura** [comment]
Good work but this has been merged.

**Pull request closed without merging**